packages/theme/src/cli/utilities/asset-ignore.ts

```diff
@@ -57,13 +58,18 @@ function filterBy(patterns: string[], type: string, invertMatch = false) {
 }
 
 export async function getPatternsFromShopifyIgnore(root: string) {
-  const shopifyIgnorePath = joinPath(root, SHOPIFY_IGNORE)
+  const localIgnore =  joinPath(root, SHOPIFY_IGNORE)
+  const globalIgnore = joinPath(os.homedir(), SHOPIFY_IGNORE)
 
-  const shopifyIgnoreExists = await fileExists(shopifyIgnorePath)
-  if (!shopifyIgnoreExists) return []
+  const patterns = await readIgnoreFile(localIgnore)
+  return patterns.concat(await readIgnoreFile(globalIgnore))
+}
 
-  const content = await readFile(shopifyIgnorePath, {encoding: 'utf8'})
+async function readIgnoreFile(path: string) {
```

Comment:
Not really a fan of this approach, must simpler to due via `flatMap` in `getPatternsFromShopifyIgnore` but given `await`'s semantics (`await` is an anti-pattern IMHO and this is just another example) it was a _cleaner_ approach 😉